samples/web/content/apprtc/bigquery/analytics_schema.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "name": "impression_name",
```

Comment:
Static member sounds OK.
Please document that the schema fields cannot be changed.